Libraries/Components/Button.js

```diff
@@ -154,6 +154,12 @@ type ButtonProps = $ReadOnly<{|
    * For arrow keys, add "ArrowLeft", "ArrowRight", "ArrowUp", "ArrowDown",
    */
   validKeysUp?: ?Array<string>,
+
+  /**
+   * The react tag of the view that follows the current view in the key view loop.
+   */
+  nextKeyViewTag?: ?number, // TODO(macOS GH#768)
+
```

Comment:
Note: Because Button is a composite component and not just a <View>, this prop doesn't actually work here. We could try setting it on <View> in line 279 instead of Touchable, but I'm more inclined to remove the prop from here altogether and only have it as a <View> prop